src/chrome/internal/sources/identifiedLoadedSource.ts

```diff
@@ -0,0 +1,78 @@
+import * as fs from 'fs';
+import { IScript } from '../scripts/script';
+import { IResourceIdentifier } from './resourceIdentifier';
+import { ILoadedSource, IScriptMapper, ICurrentScriptRelationshipsProvider as IScriptMapperProvider, ContentsLocation, SourceScriptRelationship, ImplementsLoadedSource, ScriptAndSourceMapper } from './loadedSource';
+import { ILoadedSourceToScriptRelationship } from './loadedSourceToScriptRelationship';
+import _ = require('lodash');
```

Comment:
It looks like there are types for lodash at `@types/lodash`

Reply:
test

Reply:
foo
bar

Reply:
Changed